### PR TITLE
New test: linuxrc boot from menu

### PR DESCRIPTION
**Pull request by JRivrain**
New test suite with only one simple test, to verify that booting from linuxrc with the installer's menu ("boot linux system") is working. I got rid of the 3 years old one, which was very outdated, not working, and was not in use anymore (if it ever was).
For now there is no debug logging.

- Related ticket: https://progress.opensuse.org/issues/10654
- Needles: 
https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/570
https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1184
- Verification run: 
[sle15](http://waaa-amazing.suse.cz/tests/4230)
[sle12](http://waaa-amazing.suse.cz/tests/4228)
[TW](http://waaa-amazing.suse.cz/tests/4229)

### CODE REVIEW

File: lib/bootloader_setup.pm [hunk 1]
```diff
@@ -540,6 +540,8 @@ sub select_bootmenu_more {
         send_key "ret";
     }
     send_key_until_needlematch($tag, get_var('OFW') ? 'up' : 'down', 10, 3);
+    # Redirect linuxrc logs to console when booting from menu: "boot linux system"
+    push @params, "linuxrc.log=/dev/$serialdev" if get_var('LINUXRC_BOOT');
```
Comment (rwx788) [inline lib/bootloader_setup.pm]: I believe we should reuse 2 more parameters as we have here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/bootloader_setup.pm#L406
Maybe we can simply call `bootmenu_default_params`.

**Comment by rwx788** [comment]
Needles are merged.

File: lib/bootloader_setup.pm (hunk 1 above)
Comment (JRivrain) [reply lib/bootloader_setup.pm]: It's what I was mentioning, if we use debug options such as linuxrc.debug and linuxrc.core, then we get an [extra step](http://waaa-amazing.suse.cz/tests/4219#step/boot_linuxrc/13) for kexec and then if we just type enter, boot fails in some cases.

Comment (JRivrain) [reply lib/bootloader_setup.pm]: So I thought we may do it in the scope of a follow-up ticket, as we discussed. This being said I am motivated do it right away.

Comment (rwx788) [reply lib/bootloader_setup.pm]: Ah, I missed that point. Could you please create follow up ticket and describe this difference in the behavior?

Comment (JRivrain) [reply lib/bootloader_setup.pm]: Sure

Comment (JRivrain) [reply lib/bootloader_setup.pm]: https://progress.opensuse.org/issues/54014

**Comment by rwx788** [comment]
Good job!
Please, go ahead with setting up test suites and please, update coverage in our test plan for SLES: https://gitlab.suse.de/riafarov/qa-sle-functional-y/blob/master/SLES_Integration_Level_Testplan.md

**Pull request merged**